mlflow/utils/download_cloud_file_chunk.py

```diff
@@ -41,16 +41,26 @@ def main():
             download_path=args.download_path,
             http_uri=args.http_uri,
         )
+    except requests.exceptions.ChunkedEncodingError as e:
```

Comment:
What is this error? How do you encounter this?

Reply:
This is the error hidden by the original code. We sometimes hit this error:

https://stackoverflow.com/questions/59851207/connectionreseterror-with-requests-module-raises-exception-inside-of-exception